src/systemTest/java/com/hivemq/cli/commands/cli/SubscribeST.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2019-present HiveMQ and the HiveMQ Community
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hivemq.cli.commands.cli;
+
+import com.hivemq.cli.utils.CLITestExtension;
+import com.hivemq.client.mqtt.mqtt5.Mqtt5BlockingClient;
+import com.hivemq.client.mqtt.mqtt5.Mqtt5Client;
+import com.hivemq.testcontainer.junit5.HiveMQTestContainerExtension;
+import org.jetbrains.annotations.NotNull;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Timeout;
+import org.testcontainers.utility.DockerImageName;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class SubscribeST {
+
+    private static final @NotNull HiveMQTestContainerExtension hivemq =
+            new HiveMQTestContainerExtension(DockerImageName.parse("hivemq/hivemq-ce"));
+
+    private final @NotNull CLITestExtension cliTestExtension = new CLITestExtension();
+
+    @BeforeAll
+    static void beforeAll() {
+        hivemq.start();
+    }
+
+    @AfterAll
+    static void afterAll() {
+        hivemq.stop();
+    }
```

Comment:
I think this can be achieved by using the @Container annotation on a static field

Reply:
Do you mean @RegisterExtension? I want to start HiveMQ before all tests and shut it down after all tests, not before and after each test.

Reply:
nvm with the junit 5 extension for testcontainers this would be possible, but this still uses the hivemq-testcontainer-juni5 dependency. 

We can update this in the future.